packages/paste-website/src/components/shortcodes/table-of-contents/TableOfContentsAnchor.tsx

```diff
@@ -9,10 +9,22 @@ export const TableOfContentsAnchor: React.FC<TableOfContentsAnchorProps> = ({hre
       as="a"
       href={href}
       data-cy="page-aside-anchor"
-      color="inherit"
+      color="colorTextWeak"
       textDecoration="none"
       display="inline-block"
       width="100%"
+      borderLeftColor="transparent"
+      borderLeftStyle="solid"
+      borderLeftWidth="borderWidth20"
+      _focus={{
+        outline: 'none',
+        borderLeftColor: 'colorBorderPrimaryStronger',
+        color: 'colorTextLinkStronger',
```

Comment:
if you add 

```suggestion
        color: 'colorTextLinkStronger',
        boxShadow: 'shadowFocus',
        borderRadius: 'borderRadius20',
```

you can remove the box shadow and border radius declarations from the list item styles

Reply:
Can't add the box shadow to the anchor bc it will affect the left border:

<img width="390" alt="Screen Shot 2021-11-23 at 8 41 04 AM" src="https://user-images.githubusercontent.com/8482334/143069226-9e743286-3b1a-4d02-976c-ce08f69b1c64.png">